showcase/src/app/demo/collapsible/collapsible.component.html

```diff
@@ -0,0 +1,91 @@
+<example-viewer [extensions]="['HTML', 'TS']" filesPath="collapsible/collapsible-default" exampleTitle="SHOWCASE.DEMO.COLLAPSIBLE.DEFAULT_ICON">
+	<stark-collapsible 
+			htmlId="id0" 
+			titleLabel="SHOWCASE.DEMO.COLLAPSIBLE.TITLE_0" 
+			[(isExpanded)]="collapsed[0]"
+	>
+		{{ 'SHOWCASE.DEMO.COLLAPSIBLE.HTML_CONTENT' | translate }}
+	</stark-collapsible>
+	<br/>
+	<button color="primary" (click)="toggleCollapsible(0)" mat-raised-button>{{ 'SHOWCASE.DEMO.COLLAPSIBLE.TOGGLE_COLLAPSIBLE' | translate }}</button>
+</example-viewer>
+
+<example-viewer [extensions]="['HTML', 'TS']" filesPath="collapsible/collapsible-custom1" exampleTitle="SHOWCASE.DEMO.COLLAPSIBLE.CUSTOM_ICON">
+	<stark-collapsible
+			htmlId="id1"
+			titleLabel="SHOWCASE.DEMO.COLLAPSIBLE.TITLE_1"
+			[(isExpanded)]="collapsed[1]"
+			icon="chevron-down"
+	>
+		{{ 'SHOWCASE.DEMO.COLLAPSIBLE.HTML_CONTENT' | translate }}
+	</stark-collapsible>
+	<br/>
+	<button color="primary" (click)="toggleCollapsible(1)" mat-raised-button>{{ 'SHOWCASE.DEMO.COLLAPSIBLE.TOGGLE_COLLAPSIBLE' | translate }}</button>
+	<br/>
```

Comment:
Can you add maybe a margin in a SCSS to add more space so it will be more obvious on which collapsible the button will interact with?